selene-lib/src/ast_util/visit_nodes.rs

```diff
@@ -46,6 +46,7 @@ macro_rules! make_node_visitor {
                 )+
             }
 
+            #[allow(clippy::enum_variant_names)]

```

Comment:
What's this for?

Reply:
Oh, because they're camel case...I wonder why I did that.

Reply:
Oh, I see! Fair enough.

![image](https://user-images.githubusercontent.com/3190756/135701878-a8bbf4bf-8c49-45b4-a05e-90f4ae61b52b.png)